modules/apps/oauth2-provider/oauth2-provider-scope-impl/src/main/java/com/liferay/oauth2/provider/scope/internal/liferay/ScopeLocatorImpl.java

```diff
@@ -54,7 +54,7 @@ public class ScopeLocatorImpl implements ScopeLocator {
 	public LiferayOAuth2Scope getLiferayOAuth2Scope(
 		long companyId, String applicationName, String scope) {
 
-		_jaxrsLifecycle.ensureReady();
+		JAXRSLifecycleUtil.ensureReady();
```

Comment:
Hey @Valmir-unicap, please add a breaking change note for this commit. I didn't see this before 😞